splice_machine/src/main/java/com/splicemachine/derby/jdbc/SpliceTransactionResourceImpl.java

```diff
@@ -100,9 +100,6 @@ public boolean marshallTransaction(TxnView txn) throws StandardException, SQLExc
 
 
     public void close(){
-        while(!cm.isEmpty()){
```

Comment:
New change: don't destroy ContextManager, it may still be used via an explicit  reference